test/traits.jl

```diff
@@ -224,11 +233,25 @@ end
 
     @test @inferred(eltype(HSV(30,1,0))) === Float32
     @test @inferred(eltype(HSV(30,1.0,0.0))) === Float64
+end
 
-    # eltypes_supported
-    @test N0f8 <: ColorTypes.eltypes_supported(RGB(1,0,0))
+@testset "eltype_supported/issupported" begin
+    @test Fractional <: @inferred(ColorTypes.eltypes_supported(RGB))
+    @test N0f8 <: @inferred(ColorTypes.eltypes_supported(RGB(1,0,0)))
+    @test @inferred(ColorTypes.issupported(Gray, Bool))
+    @test @inferred(!ColorTypes.issupported(AGray, Bool))
+
+    @test @inferred(ColorTypes.eltypes_supported(C2{Float32})) === Real
+    @test_broken @inferred(ColorTypes.eltypes_supported(C2A)) === Real
+
+    @test @inferred(ColorTypes.eltypes_supported(TransparentColor)) === Any
+
+    ST = @inferred(ColorTypes.eltypes_supported(StrangeGray{:X}))
+    @test_broken ST === (Normed{T} where T <: Integer)
+    @test_broken eltype(StrangeGray{UInt16}(1N14f2)) isa ST
```

Comment:
```suggestion
    @test_broken eltype(StrangeGray{UInt16}(1N14f2)) <: ST
```
but I think this will pass?

Reply:
I intended to change the `eltype` to `gray`. It seems I forgot that while doing other work. :sweat_smile:
In any case, this is a `broken` for now.